secure_redis/migrations/0001_initial.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.conf import settings
+from django.core import cache
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    def recover(apps, schema):
+        if hasattr(settings, 'DJANGO_REDIS_SECURE_CACHE_NAME'):
+            cache_name = settings.DJANGO_REDIS_SECURE_CACHE_NAME
+        else:
+            cache_name = 'default'
+        new_cache_settings = settings.CACHES.get(cache_name, {})
+        recovery_settings = new_cache_settings.get('OPTIONS', {}).get('DATA_RECOVERY')
+        if not recovery_settings:
+            return
+
+        old_key_prefix = recovery_settings.get('OLD_KEY_PREFIX')
+        old_cache_name = recovery_settings.get('OLD_CACHE_NAME')
```

Comment:
@GhalebKhaled Can you please check for those settings in `settings.py`? Raise `ImproperlyConfigured` error if not.